### PR TITLE
Allow the ability to specify the SDK version.

### PR DESCRIPTION
**Pull request by RyanAmos**
We're still currently building on the 6.1 sdk, so we needed to add this functionality.

This pull request adds the ability to specify the sdk to build with.

### CODE REVIEW

**Comment by danydev** [comment]
This is great!.

@listrophy Is there any reason because it isn't already merged on master? Just asking...:-)

File: README.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ $ bwoken test -h
         --focus                 Specify particular tests to run
         --clobber               Remove any generated file
         --skip-build            Do not build the iOS binary
+        --configuration         The build configruation to use (e.g., --configuration=Release)
+        --sdk-version           The SDK version to use (e.g., --sdk=6.1)
```
Comment (listrophy) [inline README.md]: The accompanying text should say `e.g., --sdk-verison=6.1`

File: lib/bwoken/cli.rb [hunk 2]
```diff
@@ -34,8 +34,9 @@
     on :focus=, 'Specify particular tests to run', :as => Array, :default => []
     on :clobber, 'Remove any generated file'
     on :'skip-build', 'Do not build the iOS binary'
-    on :verbose, 'Be verbose'
     on :configuration=, 'The build configruation to use (e.g., --configuration=Release)', :default => 'Debug'
+    on :'sdk-version=', 'The SDK version to use (e.g., --sdk=6.1)'
```
Comment (listrophy) [inline lib/bwoken/cli.rb]: The accompanying text should say `e.g., --sdk-verison=6.1`

**Comment by listrophy** [comment]
@RyanAmos @danydev once the two comments above are addressed, I'd be happy to pull it in.

**Comment by danydev** [comment]
@RyanAmos since you're the original developer, can you take action?. In case I'll not receive a reply (maybe you're away or something), I'll do it in a couple of days. :-)

**Comment by RyanAmos** [comment]
Pushed a commit that addresses those issues.  If you'd like it to be one commit instead, let me know.

**Pull request merged**